### ISSUE TITLE
When scrolling down, the play button disappears

### ISSUE THREAD

**Issue by wosledon** [dbcodeio/public#173]
![Image](https://github.com/user-attachments/assets/ee1377e0-de77-4dd0-8166-b9608b0403de)
When scrolling down, the play disappears, and the operation is inconvenient.

**Comment by mikeburgh** [comment]
Yeah, it’s a bit of a fudge how it works…. I use the code lens feature, and try and keep it pinned at the top, but it’s not perfect as it’s not really what that feature was designed for.

If vs code enables the ability to allow extensions to add click events to the gutter I have a play icon that stays positioned at the top of the gutter (but at the moment clicking does nothing)

I will have another look at the code lens and see if there is anything I can do to get it better. 

You can also use the play icon in the top right corner of the editor, or the keyboard shortcut ctrl/cmd enter to run commands

**Comment by mikeburgh** [comment]
I could not get this any better, as I mentioned it was a bit of a fudge.

So instead I have re worked it into the feature to better show statements and execution context.

You can see an example in #143 as part of a change to allow executing code without selections for how the codelens will work. 

So I will close this and track the feedback/feature in #143 as now they are combined into one UX item.
